### ISSUE TITLE
kbox BETA, yoursite.kbox’s server DNS address could not be found.

### ISSUE THREAD

**Issue by break9** [kalabox/kalabox#1327]
getting a `yoursite.kbox’s server DNS address could not be found.` error 

1. complete manual uninstall and trash of kbox alpha.
2. install Beta
3. download pantheon app
4. click on pantheon unicorn for the created app

browser opens http://yoursite.kbox/ but the `yoursite.kbox’s server DNS address could not be found.` error in displayed.

same result for each of the 3 apps I grabbed from pantheon.

**Comment by pirog** [comment]
Try restarting your browser

**Comment by break9** [comment]
tried restarting browser.
tried stopping app and restarting.
tried stopping app shutting down kbox restarting computer.
tried different browser.
tried deleting app and renaming it to something never used on this computer.

**Comment by pirog** [comment]
hmmm, only think i can think of then is whether the IP address of your machine was bumped. 

do this stuff and share the results

```bash
cat /etc/resolver/kbox

~/.kalabox/bin/docker-machine ls
```

**Comment by break9** [comment]
nameserver 10.13.37.100

NAME       ACTIVE   DRIVER       STATE     URL                       SWARM   DOCKER    ERRORS
Kalabox2   -        virtualbox   Running   tcp://10.13.37.100:2376           v1.11.1

**Comment by pirog** [comment]
yup that is all accurate. can you ping `10.13.37.100` and `test.kbox`?

**Comment by break9** [comment]
10.13.37.100 works

ping: cannot resolve test.kbox: Unknown host

looks like kbox froze. I am going to force quit and restart

**Comment by labboy0276** [comment]
Try restarting your computer

**Comment by break9** [comment]
did it but i will do it again

**Comment by pirog** [comment]
Can also check to make sure the proxy service is up 

```
# Drop into the VM
~/.kalabox/bin/docker-machine ssh Kalabox

# Check whether your proxy container is running
docker ps --all | grep kalabox_proxy_1

# Review its logs to look for failures 
docker logs kalabox_proxy_1
```

**Comment by break9** [comment]
so i think i cheated now it will be hard to determine the cause. I restarted the computer the ran the CLI `kbox start` and it worked as expected. : /

im glad it's working but i wish i wouldn't have done that...

**Comment by pirog** [comment]
Does start/stop work on the GUI now as well?

**Comment by break9** [comment]
i stopped the app via the CLI, then used the GUI to start and it works as expected. However, none of the files were pulled from pantheon. do you want me to create a separate issue for that?

**Comment by pirog** [comment]
If you do a Kbox pull again do you get the files as expected?

Did you get an error on the initial create? Sounds like maybe it froze and
that halted the pull?

**Comment by break9** [comment]
if i pull via the GUI or CLI i get:

`Failed after 5 retries. {"max":5,"backoff":500}: VError: code: 1err:Starting meltonair_data_1
error creating aufs mount to /mnt/sda1/var/lib/docker/aufs/mnt/3c460bd023057f60b255ff2d3ba45feb661f9e2bd66915ece841c0e2b90c8b71: invalid argument`

**Comment by break9** [comment]
I am downloading another app at the same time this might be a concurrent pantheon issue?

**Comment by break9** [comment]
i will try again once the other app is finished.

**Comment by break9** [comment]
i am going to delete everything at this point and start over from a clean Beta install

**Comment by pirog** [comment]
Very strange. You getting this on all sites? Or just this one?

Where do you have this meltonair site? Can you go into that directory and
run kbox env?

**Comment by break9** [comment]
deleted everything
re-installed kalabox BETA 
downloaded app (working in browser this time)
files were not pulled from pantheon as they should have been
GUI pull for the files only (got the files this time)

**Comment by break9** [comment]
i am getting the same file behavior on every site i install

**Comment by pirog** [comment]
I'd open a separate ticket for that and close this one out.

We test the file pull pretty robustly so I'd def be curious to figure out
what is different about your setup

**Comment by break9** [comment]
If anyone comes across this DNS issue, stoping kalabox and running `kbox start` in the app folder from terminal resolved the issue for me.
